pages/recruitment.js

```diff
@@ -0,0 +1,31 @@
+import { Header, Footer} from "@landing";
+import Developer from "landing/recruitment/Volunteer";
+import Mentor from "landing/recruitment/Mentor";
+import Communication from "landing/recruitment/DesignMarketing";
+
+export default function Recruitment() {
+    return (
+        <>
+            <Header />
```

Comment:
We should probably create a layout component that would reuse Header and Footer across all this pages that have the same base.